tutorials/equations.ipynb

```diff
@@ -40,14 +40,6 @@
    "execution_count": 1,
    "metadata": {},
    "outputs": [
-    {
```

Comment:
The changes in this jupyter notebook are not really needed. Can you revert them?

Reply:
reverted now.